code/game/objects/effects/beam.dm

```diff
@@ -185,8 +283,12 @@
 		BM.disconnect(0)
 	BM.target=AM
 	if(istype(AM))
+		if (!AM.on_moved)
```

Comment:
I'm not sure when this would be false. All `atom/movable` have this event initialised.

Reply:
That was to silence a runtime linked to beams trying to pass walls (turfs) as args. I'm not sure how that could even be possible due to the check for ismovable but you tell me.

Reply:
http://www.byond.com/docs/ref/info.html#/proc/ismovable

BYOND proc not working as intended, it seems.